CHANGELOG.md

```diff
@@ -38,20 +38,45 @@ This text is announced by screen readers when the character count input is focus
 
 This was added in [pull request #2742: Add ability to customise character count fallback text](https://github.com/alphagov/govuk-frontend/pull/2742).
 
+#### Localise the character count's counter message
+
+You can now translate the text shown by the [character count](https://design-system.service.gov.uk/components/character-count/) component to inform users of:
+
+- when they reached the maximum number of characters or words
+- the number of characters or words over or under that maximum
+
+The Nunjucks macro accepts new options to let you customise each message:
+
+- You can use `charactersAtLimitText` or `wordsAtLimitText` to provide the text to show when users reached the limit.
+- When users are under or over the limit, the component will pluralise the message according to the number of characters or words remaining, and the configured locale. You can pass `charactersUnderLimitText` or `wordsUnderLimitText` (when under the limit) and `charactersOverLimitText` or `wordsOverLimitText` (when over the limit) an object with the different plural forms expected by your locale. You'll find guidance about [the plural forms in our documentation about localising GOV.UK Frontend](https://design-system.service.gov.uk/get-started/localisation/#pluralisation). The component will replace `%{count}` with the number of characters over or under the limit.
```

Comment:
```suggestion
- When users are under or over the limit, the component will pluralise the message according to the configured locale and the number of characters or words remaining. When under the limit you can pass `charactersUnderLimitText` or `wordsUnderLimitText` and when over the limit you can pass `charactersOverLimitText` or `wordsOverLimitText`  to an object with the different plural forms expected by your locale. You'll find guidance about [the plural forms in our documentation about localising GOV.UK Frontend](https://design-system.service.gov.uk/get-started/localisation/#pluralisation). The component will replace `%{count}` with the number of characters over or under the limit.
```

Reply:
I don't think we're putting the localisation info in the design system. I think the URL needs to be to the frontend docs (new page).

Reply:
True, we'll be doing a PR to replace the links in the changelog when we'll have decided on URLs, as there are some in the accordion section as well (maybe the configuration options too). 

Regarding the content, I think the meaning changed a bit with your update. It makes it sound like users have to take an action "When under the limit" and "when over the limit". Actually, it's the component that'll make the decision to use the `...UnderLimitText` or `...OverLimitText` depending of whether the value entered is under or over the limit. Users only need to pass the keys for their message. 

In hindsight, that list item might be trying to group too many things by going for both scenarios at once. Maybe it'll work better if it's split up a bit?

```suggestion
- Use `charactersUnderLimitText` or `wordsUnderLimitText` to provide the text that shows when users are under the limit. The component will pluralise the message according to the configured locale and the number of characters or words remaining.
- Use `charactersOverLimitText` or `wordsOverLimitText` to provide the text that shows when users are over the limit. The component will pluralise the message according to the configured locale and the number of characters or words remaining.  

You'll find guidance about [the plural forms in our documentation about localising GOV.UK Frontend](https://design-system.service.gov.uk/get-started/localisation/#pluralisation). The component will replace `%{count}` with the number of characters over or under the limit.
```

That makes it a little repetitive and very close to what will be in the Nunjucks option table, but that may not be a bad thing? I'm open to other ideas

Reply:
Yes, this is clearer. 👍🏻